packages/compass-generative-ai/src/atlas-ai-service.ts

```diff
@@ -395,6 +467,60 @@ export class AtlasAiService {
     );
   }
 
+  /**
+   * @returns {MockDataSchemaResponse} which contains faker.js mappings used to produce a
+   * faker.js factory function for the purposes of generating mock document data.
+   *
+   * @throws {AtlasAiServiceApiResponseParseError} when the response cannot be parsed into the expected schema.
+   */
+  async getMockDataSchema(
+    input: MockDataSchemaRequest,
+    connectionInfo: ConnectionInfo
+  ): Promise<MockDataSchemaResponse> {
+    const { collectionName, databaseName } = input;
+    let schema = input.schema;
+
+    const url = `${this.getUrlForEndpoint('mock-data-schema', connectionInfo)}`;
```

Comment:
This already returns a string

```suggestion
    const url = this.getUrlForEndpoint('mock-data-schema', connectionInfo);
```